### ISSUE TITLE
Why use synchronized BatchNorm?

### ISSUE THREAD

**Issue by alievk** [AliaksandrSiarohin/first-order-model#350]
Hi Aliaksandr,

What's the point to use SynchronizedBatchNorm2d instead of vanilla BatchNorm2d?

**Comment by AliaksandrSiarohin** [comment]
In my experience it usually works better, so I start with sync bn. I did not test it with the latest model though, so not sure if it better in this case.

**Comment by alievk** [comment]
You mean it generally works better for training deep learning models or particularly for FOMM?

**Comment by AliaksandrSiarohin** [comment]
In general.

**Comment by alievk** [comment]
I only see a difference in equivariance_jacobian (pink w/ SyncBatchNorm, green w/o)

![image](https://user-images.githubusercontent.com/4682236/103149458-b00a9d80-477a-11eb-81da-bf19f1770fc4.png)

However, SyncBatchNorm is slow and memory consuming.

**Comment by alievk** [comment]
Did you explore the effect of the learning rate on the KP detector? Is the value defined in the config optimal?

**Comment by AliaksandrSiarohin** [comment]
Not really.
